.github/workflows/_test.yml

```diff
@@ -44,14 +44,6 @@ jobs:
         shell: bash
         run: make test
 
-      - name: Install integration dependencies
-        shell: bash
-        run: poetry install --with=test_integration
-
-      - name: Check integration tests compile
-        shell: bash
-        run: poetry run pytest -m compile tests/integration_tests
-
```

Comment:
@baskaryan to confirm removing this is ok (it's still in _compile_integration_test.yml, which is called for langchain and experimental)